client/cypress/e2e/spec.cy.ts

```diff
@@ -1,6 +1,6 @@
 /// <reference types="cypress" />
-describe('Carousel', () => {
-  it('Visits the PokemonAapp Page', () => {
+describe('Loading String Test', () => {
```

Comment:
will we left this file when we have carousel.cy.ts in landingPageTests? or should we remove it?